sonar-project.properties

```diff
@@ -1,3 +1,11 @@
-sonar.projectKey=centreon-dsm-20.10
-sonar.projectName=Centreon DSM 20.10
+# project
+sonar.projectKey={PROJECT_TITLE}
+sonar.projectName={PROJECT_NAME}
+sonar.projectVersion={PROJECT_VERSION}
 sonar.sources=.
+
+#exclusions
+sonar.coverage.exclusions=  bin/**/*,\
+                            cron/**/*,\
+                            doc/**/*,\
```

Comment:
seems not consistent with awie : https://github.com/centreon/centreon-awie/pull/45/files
any reason to put `doc` in `sonar.coverage.exclusions` instead of `sonar.exclusions` directly ?